### PR TITLE
[ticket/14807] Updates dependencies

### PR DESCRIPTION
**Pull request by Nicofuma**
Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [ ] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

paragonie/random_compat: Updated to 2.x, BC break: removal of OpenSSL support
s9e/text-formatter: Updated to 0.7.x, BC break not affecting us

```
$ composer update symfony/browser-kit  symfony/config symfony/console symfony/css-selector symfony/debug symfony/dependency-injection symfony/dom-crawler symfony/event-dispatcher symfony/filesystem symfony/finder symfony/http-foundation symfony/http-kernel symfony/process symfony/proxy-manager-bridge symfony/routing symfony/twig-bridge symfony/yaml paragonie/random_compat patchwork/utf8 s9e/text-formatter
Loading composer repositories with package information
Updating dependencies (including require-dev)
  - Removing symfony/filesystem (v2.8.9)
  - Installing symfony/filesystem (v2.8.11)
    Downloading: 100%         

  - Removing symfony/config (v2.8.9)
  - Installing symfony/config (v2.8.11)
    Downloading: 100%         

  - Removing symfony/console (v2.8.9)
  - Installing symfony/console (v2.8.11)
    Downloading: 100%         

  - Removing symfony/finder (v2.8.9)
  - Installing symfony/finder (v2.8.11)
    Loading from cache

  - Removing symfony/debug (v2.8.9)
  - Installing symfony/debug (v2.8.11)
    Downloading: 100%         

  - Removing symfony/http-foundation (v2.8.9)
  - Installing symfony/http-foundation (v2.8.11)
    Downloading: 100%         

  - Removing symfony/event-dispatcher (v2.8.9)
  - Installing symfony/event-dispatcher (v2.8.11)
    Loading from cache

  - Removing symfony/http-kernel (v2.8.9)
  - Installing symfony/http-kernel (v2.8.11)
    Downloading: 100%         

  - Removing symfony/dependency-injection (v2.8.9)
  - Installing symfony/dependency-injection (v2.8.11)
    Downloading: 100%         

  - Removing symfony/proxy-manager-bridge (v2.8.9)
  - Installing symfony/proxy-manager-bridge (v2.8.11)
    Loading from cache

  - Removing symfony/routing (v2.8.9)
  - Installing symfony/routing (v2.8.11)
    Downloading: 100%         

  - Removing symfony/twig-bridge (v2.8.9)
  - Installing symfony/twig-bridge (v2.8.11)
    Downloading: 100%         

  - Removing symfony/yaml (v2.8.9)
  - Installing symfony/yaml (v2.8.11)
    Loading from cache

  - Removing symfony/dom-crawler (v2.8.9)
  - Installing symfony/dom-crawler (v2.8.11)
    Loading from cache

  - Removing symfony/browser-kit (v2.8.9)
  - Installing symfony/browser-kit (v2.8.11)
    Downloading: 100%         

  - Removing symfony/css-selector (v2.8.9)
  - Installing symfony/css-selector (v2.8.11)
    Downloading: 100%         

  - Removing symfony/process (v2.8.9)
  - Installing symfony/process (v2.8.11)
    Downloading: 100%         

  - Removing patchwork/utf8 (v1.1.31)
  - Installing patchwork/utf8 (v1.3.1)
    Downloading: 100%         

  - Removing s9e/text-formatter (0.6.2)
  - Installing s9e/text-formatter (0.7.1)
    Downloading: 100%         

  - Removing paragonie/random_compat (v1.4.1)
  - Installing paragonie/random_compat (v2.0.2)
    Loading from cache

Writing lock file
Generating autoload files
> echo 'You MUST manually modify the clean-vendor-dir target in build/build.xml when adding or upgrading dependencies.'
You MUST manually modify the clean-vendor-dir target in build/build.xml when adding or upgrading dependencies.
```

Tracker ticket:
https://tracker.phpbb.com/browse/PHPBB3-14807

### CODE REVIEW

**Comment by rxu** [comment]
@Nicofuma We're going to get s9e/text-formatter 0.8.0 next weekend which is required for https://github.com/phpbb-extensions/mediaembed :smiley: 
See https://github.com/s9e/TextFormatter/issues/45#issuecomment-250946463

**Comment by marc1706** [comment]
There seems to be one test failure caused by changing from twemoji to emojione. Not sure if this is config related.

**Comment by VSEphpbb** [comment]
LOL. I talked with @marc1706 about updating s9e/textformatter actually 0.8 for the 3.2.x branch (but @JoshyPHP  won't be releasing it until next weekend).

**Comment by Nicofuma** [comment]
No worries, we can update it again later :)

**Comment by JoshyPHP** [comment]
@marc1706 Right, the default image set was changed to EmojiOne. You can [explicitly select the Twemoji set](http://s9etextformatter.readthedocs.io/Plugins/Emoji/Synopsis/#using-the-twemoji-set) if you prefer but I'd recommend keeping the default. I'm having some troubles with Twemoji's naming convention that will only get more complicated with future Unicode releases.

@VSEphpbb @Nicofuma You can always ping me for an earlier release if it lines up better with yours. It only takes a couple of key presses for me but I try not to spam releases if I don't have to.

**Comment by Nicofuma** [comment]
@JoshyPHP I'm not in a hurry, as long as you release in the following weeks that's fine with me :)

**Comment by JoshyPHP** [comment]
@Nicofuma I released 0.8.0 today if you want to bump the dependency.

**Comment by Nicofuma** [comment]
OK thanks

**Comment by Nicofuma** [comment]
Issue on travis: https://github.com/s9e/TextFormatter/issues/46

File: phpBB/phpbb/template/twig/loader.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
 */
 
 namespace phpbb\template\twig;
+use Twig_Loader_Filesystem;
```
Comment (marc1706) [inline phpBB/phpbb/template/twig/loader.php]: Needs a blank line after the namespace.

Comment (Nicofuma) [inline phpBB/phpbb/template/twig/loader.php]: fixed

**Pull request merged**